README.md

```diff
@@ -138,7 +138,9 @@ pip install -e .
 ## Example Usage
 
 Example usage of `idrstream` can be found at [example.ipynb](example.ipynb).
-This notebook was run as a python script ([example.py](example.py)).
+This notebook was run as a python script ([example.py](example.py)) and hence has no output for the final cell.
+
+**Note**: `idrstream` can be used to extract object oulines as extra metadata by passing `extra_metadata=["object_outlines"]` during `idrstream.run_stream()`. Similarly, the desired batch numbers can be chosen with `batch_nums=[#,#,#]`.
```

Comment:
```suggestion
**Note**: You can use `idrstream` o extract object oulines as extra metadata by passing `extra_metadata=["object_outlines"]` during `idrstream.run_stream()`.
Similarly, you can choose the desired batch numbers with `batch_nums=[#,#,#]`.
```

Shifting to active voice (no passive voice, please!). I also split the multi-sentence line to two lines.

Reply:
Thank you!